chrome/content/zotero/zoteroPane.js

```diff
@@ -1305,6 +1305,11 @@ var ZoteroPane = new function()
 						var type = mru ? mru.split(',')[0] : 'book';
 						await ZoteroPane.newItem(Zotero.ItemTypes.getID(type));
 						let itemBox = document.getElementById('zotero-editpane-item-box');
+						// If the info pane is collapsed, focus the title in the header
+						if (!itemBox._section.open) {
```

Comment:
It would be better not to rely on a private property — we do that in large part so that we can feel comfortable changing things within the same file without checking for other references. If this isn't available on a public property, we should expose it.